api/graphql/schema.graphql

```diff
@@ -99,8 +106,16 @@ type Membership {
   lastViewedAt: String
   newPostCount: Int
   community: Community
+  settings: MembershipSettings
 }
 
+type MembershipSettings {
+  send_push_notifications: Boolean
```

Comment:
I wasn't sure whether to change these or the user settings to camel case. I know we've done that elsewhere, but this would be adding a little extra complexity in a bunch of places and not sure what it buys us.